nss-day-cohort-47/holiday-road-salt-lake-skaters
# all details buttons populate to the DOM

# Description

Added event listeners for parks and attractions. 

Fixes # (issue)

## Type of change



# Testing Instructions

Select park and attraction from the dropdown menus. Click details button. Details should populate on the DOM. 

# Checklist:
